.github/workflows/build.yml

```diff
@@ -56,7 +56,78 @@ jobs:
         run: |
           .\build.ps1 build -Configuration ${{ env.CONFIGURATION }} -Version ${{ env.BUILD_VERSION }} -BuildCounter ${{ github.run_number }}
         shell: pwsh
+        
```

Comment:
Below you'll see a bunch of individual uploads of "sarif" files. These are log files with build warnings in them. However, there is a small bug when you run in GitHub Actions: the sarif files have a blank language attribute. This prevents us from being able to upload the files. So the first step below fixes that in all sarif files.